src/app/CommandHandler.cpp

```diff
@@ -315,6 +315,18 @@ CHIP_ERROR CommandHandler::ProcessCommandDataIB(CommandDataIB::Parser & aCommand
         return AddStatus(concretePath, Protocols::InteractionModel::Status::NeedsTimedInteraction);
     }
 
+    if (CommandIsFabricScoped(concretePath.mClusterId, concretePath.mCommandId))
```

Comment:
Worth documenting in ProcessGroupCommandDataIB why the check is not needed there (probably because if we found a group key we know we have a fabric?) or doing the check there too.